connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorSourceConnector.java

```diff
@@ -99,6 +99,7 @@ public void start(Map<String, String> props) {
             sourceAdminClient = AdminClient.create(config.sourceAdminConfig());
             targetAdminClient = AdminClient.create(config.targetAdminConfig());
         }
+        MirrorUtils.createTopic(config.offsetSyncsTopic(), (short) 1, config.internalTopicReplicationFactor(), config.targetAdminConfig());
```

Comment:
If we do this with `scheduler.execute` like `createTopicPartitions` below, we get better exception handling, logging, and timing info for free.